static/app/components/sidebar/newOnboardingStatus.tsx

```diff
@@ -73,12 +73,11 @@ export function NewOnboardingStatus({
   });
 
   const label = walkthrough ? t('Guided Tours') : t('Onboarding');
-  const totalRemainingTasks = allTasks.length - doneTasks.length;
   const pendingCompletionSeen = doneTasks.length !== completeTasks.length;
+  const allTasksCompleted = allTasks.length === completeTasks.length;
```

Comment:
It’s better to check for `completeTasks` instead of `doneTasks`, as `completeTasks` are the ones that have already been seen by the user